commit 9c3a46e53ebfff376eefee88c2b8745e17bdc21b
Author:     Günther Deschner <gd@samba.org>
AuthorDate: Tue Feb 4 16:38:46 2014 +0100
Commit:     Andreas Schneider <asn@samba.org>
CommitDate: Mon Mar 10 17:18:14 2014 +0100

    PATCHSET6 librpc/nbt: increase MAX_COMPONENTS limit for nbt_names.
    
    domains with more then 10 subdomains are not so uncommon.
    
    https://bugzilla.samba.org/show_bug.cgi?id=10439
    
    Guenther
    
    Signed-off-by: Günther Deschner <gd@samba.org>
    Reviewed-by: Andreas Schneider <asn@samba.org>
    
    (cherry picked from commit 4e05bad0d18e351cb2a2db74860e77adea727c79)
---
 libcli/nbt/nbtname.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libcli/nbt/nbtname.c b/libcli/nbt/nbtname.c
index fec8e8e..3aa0000 100644
--- a/libcli/nbt/nbtname.c
+++ b/libcli/nbt/nbtname.c
@@ -30,7 +30,7 @@
 #include "lib/util/util_net.h"
 
 /* don't allow an unlimited number of name components */
-#define MAX_COMPONENTS 10
+#define MAX_COMPONENTS 128
 
 /**
   print a nbt string
